src/screens/NewsScreen/index.js

```diff
@@ -271,7 +271,7 @@ const NewsScreen = () => {
         screen="news_screen"
         getValueBlock={onBlockedDomain}
       />
-    </SafeAreaProvider>
+    </View>
   );
 };
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the closing tag should be `</SafeAreaProvider>` instead of `</View>`, unless you have a specific reason for this change and have tested it across different devices.

```diff
- </View>
+ </SafeAreaProvider>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->